### PR TITLE
Ledger -> Ledger Nano S

### PR DESCRIPTION
**Pull request by digityler**
As per Ledger documentation guidelines, changed "Ledger" to either "Ledger Nano S", "Ledger device", or "Ledger hardware wallet"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,9 +81,9 @@ More advanced techniques for those interested in even more layers of security
 or plausible deniability features should look at the
 [Ledger's documentation on this](https://support.ledgerwallet.com/hc/en-us/articles/115005214529-Advanced-Passphrase-options).
 
-Note that Ledgers with different seeds will appear to `tezos-client` to be
+Note that Ledger hardeware wallets with different seeds will appear to `tezos-client` to be
```
Comment (mikereinhart) [inline README.md]: Spelling error on hardeware

File: README.md [hunk 2]
```diff
@@ -530,7 +530,7 @@ encryption system (`ed25519`, `secp256k1`, or P-256). The second three show you
 examples of how to append a BIP32 path to the root key to construct a new
 address.
 
-The Ledger does not currently support non-hardened path components. All
+The Ledger Nano Sdoes not currently support non-hardened path components. All
```
Comment (mikereinhart) [inline README.md]: needs space

**Comment by digityler** [comment]
looks good

**Comment by matthewbauer** [comment]
Thanks for the contribution. It looks like these changes have been addressed in master, and this PR is no longer necessary. Feel free to reopen if that is not the case!

**Pull request closed without merging**